packages/dev/core/src/Materials/standardMaterial.ts

```diff
@@ -878,6 +878,10 @@ export class StandardMaterial extends PushMaterial {
      * @returns a boolean specifying if alpha blending is needed
      */
     public override needAlphaBlending(): boolean {
+        if (this._hasTransparencyMode) {

```

Comment:
Adding a comment to avoid merging it before conversation about deprecation is concluded.